pkgs/by-name/ko/koboldcpp/package.nix

```diff
@@ -65,8 +62,10 @@ effectiveStdenv.mkDerivation (finalAttrs: {
   buildInputs =
     [ tk ]
     ++ finalAttrs.pythonInputs
-    ++ lib.optionals stdenv.hostPlatform.isDarwin [ apple-sdk_12 ]
-    ++ lib.optionals openblasSupport [ openblas ]
+    ++ lib.optionals stdenv.hostPlatform.isDarwin [
+      apple-sdk_12
+      (darwinMinVersionHook "10.15")
```

Comment:
this isn't needed on master anymore.. both x64 and aarch64 now default to 11.

Reply:
Thanks for letting me know. When I made this PR initially, this wasn't the case, those were docs:

https://github.com/NixOS/nixpkgs/blob/62501808a721f34d429f0bde2adb9ddcdda66f89/doc/stdenv/platform-notes.chapter.md?plain=1#L92-L103

Reply:
np. tho if you are planning on back porting this into release-24.11 it is still needed there as the x64 darwin sdk is still at 10.12 and minVersion is 10.12 as well.